src/backend/utils/logger.js

```diff
@@ -3,7 +3,7 @@ const expressPino = require('express-pino-logger');
 const os = require('os');
 const path = require('path');
 
-require('../lib/config');
+require('dotenv').config();
```

Comment:
We do this in [lib/config.js](https://github.com/Seneca-CDOT/telescope/blob/master/src/backend/lib/config.js) already, as well as some fallback logic.

I'd like to isolate the config loading to `lib/config.js` vs. doing it here.  Can you see what we're doing wrong in that file instead of this change?

Reply:
Yea, I am going to look it

Reply:
In config.js file. It works as long as` const result = dotenv.config(); ` before `const { logger } = require('../utils/logger');`

Reply:
Ah, interesting, because we have a circular dependency here.

Let's rip the `logger` dependency out of `lib/config.js` and use `console.log()` in that file instead.  Maybe we can avoid logging in there altogether.